### DISCUSSION TITLE
Obtain accuracy

### DISCUSSION THREAD

**Discussion by zhangxin702** [Autostronomy/AstroPhot#113] [Q&A] [answered]
Hello, I have a question. How can one determine the accuracy of a fitted model?Alternatively, which other software can I use to assess the accuracy of the model fitted with this software?

**Comment by ConnorStoneAstro** [comment, marked as answer]
Hi @zhangxin702 , it is possible to recover a full covariance matrix giving the uncertainty in every parameter. There are examples in the GettingStarted and FittingMethods notebooks. However, these uncertainties are only valid if the variance map provided for the image is valid, and if the model is a reasonable representation of the galaxy. Assessing those conditions is typically more up to the astronomer than an automated tool.
